clash-cores/src/Clash/Cores/SPI.hs

```diff
@@ -254,9 +304,57 @@ spiMaster
   -- 4. Busy signal indicating that a transmission is in progress, new words on
   --    the data line will be ignored when /True/
   -- 5. (Maybe) the word send from the slave to the master
-spiMaster mode fN fW din miso =
+spiMaster cfg fN fW din miso =
```

Comment:
The first argument is an `SPIMode` which was previously called `mode` and in the new `spiMaster'` is also called `mode`. I think this `cfg` here accidentally snuck in, could you rename in `mode`?